4.8/Dockerfile

```diff
@@ -6,17 +6,15 @@ RUN groupadd --gid 1000 node \
 # gpg keys listed at https://github.com/nodejs/node
 RUN set -ex \
   && for key in \
-    9554F04D7259F04124DE6B476D5A82AC7E37093B \
```

Comment:
@chorrell I've moved into two lists

```
 # Current keys from https://github.com/nodejs/node#release-tea
 # Older keys from https://github.com/nodejs/node#release-tea
```

`9554F04D7259F04124DE6B476D5A82AC7E37093B` is present in the later.